JapaneseFilter/sections/specific.txt

```diff
@@ -40,6 +40,12 @@ yutura.net,goetheweb.jp,mono-money.com,tetsudo-ch.com,home.kingsoft.jp,coco-tte.
 !
 !
 seijiyama.jp###ad-two-center
+hotokami.jp##.shitennoji-banner-js
+hotokami.jp##.shitennoji-big-banner
+hotokami.jp##.layout_fix_advice--apps
```

Comment:
Please move this to AdGuard Annoyances - self-promo.

Reply:
ok,done